### PR TITLE
Tools partial

### PR DESCRIPTION
**Pull request by ashleykolodziej**
This pull request creates a new partial, `burf-tools`, with absolutely minimal dependencies and CSS output. Perfect for developing a static site or plugin where you already have opinionated CSS and a normalize framework, and don't need the extra overhead.

One of the main goals of this partial is to output absolutely no CSS unless it's explicitly used, so you will not see anything besides mixins and placeholder classes in here. There is also an assumption that you'll be doing minimal overrides if you import this package, so - unlike a theme - you're going to set all your variables before importing this partial if you're going to change anything.

### CODE REVIEW

File: css-dev/burf-base.scss [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 // dependencies and resets
 @import "normalize/import-now";
-@import "math";
+@import "node_modules/normalize-scss/sass/normalize/import-now";
+@import "node_modules/mathsass/dist/math";
```
Comment (toddmilliken) [inline css-dev/burf-base.scss]: I think we want to use the tilde`~` style imports here for consistency. 

```suggestion
@import "~/mathsass/dist/math";
```

File: css-dev/burf-base.scss [hunk 2]
```diff
@@ -4,7 +4,8 @@
 
 // dependencies and resets
 @import "normalize/import-now";
-@import "math";
+@import "node_modules/normalize-scss/sass/normalize/import-now";
```
Comment (toddmilliken) [inline css-dev/burf-base.scss]: I think we want to use the tilde`~` style imports here for consistency. 

```suggestion
@import "~/normalize-scss/sass/normalize/import-now";
```

File: css-dev/burf-tools/_tools-grunt.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+// pre-existing framework or environment with its own normalizing
+// partials and other opinionated stuff.
+//
+// =================================================================
+
+// =================================================================
+// Dependencies
+// =================================================================
+
+@import "math";
```
Comment (toddmilliken) [inline css-dev/burf-tools/_tools-grunt.scss]: I think we also need to add the node modules import notation here as well, if we're avoiding using includePaths in our grunt config:

```suggestion
@import "~/mathsass/dist/math";
```

File: css-dev/burf-base.scss (hunk 1 above)
Comment (ashleykolodziej) [reply css-dev/burf-base.scss]: This works for webpack and CGB, but unfortunately doesn't seem to work with grunt-sass, so I think it has to stay as is.

File: css-dev/burf-tools/_tools-grunt.scss (hunk 3 above)
Comment (ashleykolodziej) [reply css-dev/burf-tools/_tools-grunt.scss]: Same deal with grunt-sass over here - it won't recognize the `~` so we're stuck using the folder path name for now.

**Pull request merged**